package.json

```diff
@@ -630,7 +630,7 @@
 		"font-awesome": "^4.7.0",
 		"gulp": "^4.0.2",
 		"lodash.throttle": "4.1.1",
-		"mathjax": "^2.7.9",
```

Comment:
This is a major version and the built-in converter is not compatible with it. Since Asciidoctor is still using MathJax 2.x I would prefer to stick with it for now.

Reply:
I'll revert it since 2.7.9 is the latest 2.x for MathJax. Before creating the PR I ran the tests without any issue. Maybe I did something wrong or the tests don't cover this scenario.

Reply:
We do not have a good test coverage right now but this is mostly something you will notice on the preview/client side since the STEM expressions won't be rendered properly.